spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1846,9 +1846,13 @@ def _do_run(self, n_machine_time_steps, loading_done, run_until_complete):
             # if in debug mode, do not shut down machine
             in_debug_mode = self._config.get("Mode", "mode") == "Debug"
             if not in_debug_mode:
-                self.stop(
-                    turn_off_machine=False, clear_routing_tables=False,
-                    clear_tags=False)
+                try:
+                    self.stop(
+                        turn_off_machine=False, clear_routing_tables=False,
+                        clear_tags=False)
+                except Exception:
+                    logger.error("Error when attempting to stop")
+                    traceback.print_exc()
```

Comment:
Please write this to the log (at high-level priority) rather than just printing it out. Perhaps by combining with the previous line like this:
```py
logger.error("Error when attempting to stop", exc_info=True)
```